src/chromium/policy.md

```diff
@@ -0,0 +1,22 @@
+# Chromium Rust policy
+
+Chromium does not yet allow first-party Rust except in rare cases as approved
+by Chromium's [Area Tech Leads](https://source.chromium.org/chromium/chromium/src/+/main:ATL_OWNERS).
+
+Chromium's policy on third party libraries is outlined [here](https://chromium.googlesource.com/chromium/src/+/main/docs/adding_to_third_party.md#rust) -
+Rust is allowed for third party libraries under various circumstances, including
+if they're the best option for performance or for security.
+
+Very few Rust libraries directly expose a C/C++ API, so that means that nearly
+all such libraries will require a small amount of first-party glue code.
+
+> First-party Rust glue code for a particular third-party crate should
+> normally be kept in `third_party/rust/<crate>/<version>/wrapper`.
+
+Because of this, today's course will be heavily focused on:
+
+* Bringing in third-party Rust code ("crates")
```

Comment:
nit/suggestion: s/ third-party Rust code / third-party Rust libraries /
(I think it reads better + more directly clarifies that a Rust "crate" is a Rust "library").

Reply:
Done